crates/common_utils/src/pii.rs

```diff
@@ -152,7 +152,8 @@ mod pii_masking_strategy_tests {
     #[test]
     fn test_invalid_card_number_masking() {
         let secret: Secret<String, CardNumber> = Secret::new("1234567890".to_string());
-        assert_eq!("123456****", format!("{secret:?}"));
+        println!("//////////////////////////////////////{:?}", secret);
```

Comment:
remove this, if not required

Reply:
Done manually added `*** alloc::string::String ***` as its for invalid card nos